### PR TITLE
[openhabcloud] Further README improvements

### PR DESCRIPTION
**Pull request by florian-h05**
After seeing the results of #16941 on the website, I noticed a few things that could be improved.

### CODE REVIEW

**Comment by florian-h05** [comment]
@jimtng Would be nice if you could add some jRuby examples for the cloud notification actions to the README as well.

File: bundles/org.openhab.io.openhabcloud/README.md [hunk 1]
```diff
@@ -133,9 +135,9 @@ The additional parameter for these variants have the following meaning:
 - `title`: The title of the notification. Defaults to "openHAB" inside the Android and iOS apps.
 - `onClickAction`: The action to be performed when the user clicks on the notification. Specified using the [action syntax](#action-syntax).
 - `mediaAttachmentUrl`: The URL of the media attachment to be displayed with the notification. This URL must be reachable by the push notification client.
-- `actionButton1`: The action to be performed when the user clicks on the first action button. Specified as `Titel=$action`, where `$action` follow the [action syntax](#action-syntax).
-- `actionButton2`: The action to be performed when the user clicks on the second action button. Specified as `Titel=$action`, where `$action` follow the [action syntax](#action-syntax).
-- `actionButton3`: The action to be performed when the user clicks on the third action button. Specified as `Titel=$action`, where `$action` follow the [action syntax](#action-syntax).
+- `actionButton1`: The action to be performed when the user clicks on the first action button. Specified as `Titel=$action`, where `$action` follows the [action syntax](#action-syntax).
```
Comment (jlaur) [inline bundles/org.openhab.io.openhabcloud/README.md:138]: I don't fully understand the syntax here, but should "Titel" be "Title"?

File: bundles/org.openhab.io.openhabcloud/README.md [hunk 2]
```diff
@@ -210,15 +214,18 @@ end
 
 ```javascript
 rules.when().item('Apartment_Window').changed().to('OPEN').then(() => {
-  actions.notificationBuilder('Apartment window was opened!').withIcon('window').withSeverity('HIGH').send();
+  actions.notificationBuilder('Apartment window was opened!')
+          .withIcon('window')
```
Comment (jlaur) [inline bundles/org.openhab.io.openhabcloud/README.md:218]: Is the indentation correct here? In the line above, two spaces are used. So with eight spaces in this line, three levels deeper?

File: bundles/org.openhab.io.openhabcloud/README.md [hunk 3]
```diff
@@ -229,7 +236,9 @@ rule "Motion Detected Notification"
 when
   Item Apartment_MotionSensor changed to ON
 then
-  sendBroadcastNotification("Motion detected in the apartment!", "motion", "MEDIUM", "Motion Detected", null, "https://apartment.my/camera-snapshot.jpg", "command:Apartment_Light:ON", null, null)
+  sendBroadcastNotification("Motion detected in the apartment!", "motion", "MEDIUM",
+                                    "Motion Detected", null, "https://apartment.my/camera-snapshot.jpg",
```
Comment (jlaur) [inline bundles/org.openhab.io.openhabcloud/README.md:240]: This indentation also looks strange?

File: bundles/org.openhab.io.openhabcloud/README.md (hunk 1 above)
Comment (digitaldan) [reply bundles/org.openhab.io.openhabcloud/README.md:138]: It should, just a typo

**Comment by digitaldan** [comment]
@florian-h05 thanks again.  I totally forgot about updating the README in my haste.

**Comment by jimtng** [comment]
@florian-h05 I created a PR on your repo to add jruby examples. The ruby support is added in https://github.com/openhab/openhab-jruby/pull/285

File: bundles/org.openhab.io.openhabcloud/README.md (hunk 3 above)
Comment (florian-h05) [reply bundles/org.openhab.io.openhabcloud/README.md:240]: This intends all params in one line

**Comment by florian-h05** [comment]
@jlaur Can you please take another look?

**Review by jlaur — APPROVED** [review]
LGTM

**Pull request merged**